pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	"k8s.io/klog/v2"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
looks like this file doesn't need to be changed

Reply:
removed.